Offroad-iOS/Offroad-iOS/Presentation/MyPage/Title/View/Cell/CollectedTitleCollectionViewCell.swift

```diff
@@ -92,10 +103,18 @@ extension CollectedTitleCollectionViewCell {
     
     //MARK: - Func
     
-    func configureCell(data: CollectedTitleModel) {
-        titleLabel.text = data.titleString
-        collectConditionLabel.text = "\(data.collectConditionString) 달성 시 획득"
-        collectConditionLabel.highlightText(targetText: data.collectConditionString, font: .offroad(style: .iosTextContents))
-        lockedView.isHidden = data.isCollected ? false : true
+    func configureCollectedCell(data: EmblemDataList) {
+        titleLabel.text = data.emblemName
+        collectConditionLabel.text = "\(data.clearConditionQuestName) 달성 시 획득"
+        collectConditionLabel.highlightText(targetText: data.clearConditionQuestName, font: .offroad(style: .iosTextContents))
+        newTagImageView.isHidden = data.isNewGained ? false : true
+        lockedView.isHidden = true
+    }
+    
+    func configureNotCollectedCell(data: EmblemDataList) {
+        titleLabel.text = data.emblemName
+        collectConditionLabel.text = "\(data.clearConditionQuestName) 달성 시 획득"
+        collectConditionLabel.highlightText(targetText: data.clearConditionQuestName, font: .offroad(style: .iosTextContents))
     }
```

Comment:
P1
잠금을 해제하지 않은 셀이 재사용될 때, 잠가야 하는 셀의 configureNotCollectedCell(data:) 메서드가 호출되더라도 다시 잠그는 코드가 없어, 획득하지 않은 셀임에도 획득한 것으로 뜨는 이슈가 발생합니다. 
따라서 아래의 방법처럼 획득하지 않은 칭호의 cell 을 구성할 때 `lockedView`의 `isHidden` 속성을 `false` 로 명시해 주는 코드를 추가하는 것을 제안합니다!

```suggestion
        collectConditionLabel.highlightText(targetText: data.clearConditionQuestName, font: .offroad(style: .iosTextContents))
        lockedView.isHidden = false
    }
```